docs/manual/manual_velocity_control_mechanisms.rst

```diff
@@ -0,0 +1,419 @@
+supply. Therefore, a *PWM Command* of 0.1 should result in the same spin velocity as a *Voltage Command* of 1.5V. You will see this sample 
+prints two statements. First, the observed drive velocity when driven by a *Voltage Command* of 1.5V, and the second, the observed drive 
+velocity when driven by a *PWM Command* of 0.1. In the PWM case, you will also see the voltage observed at the module's input.  We also 
+set the module's :ref:`timeout period <manual_timeout>` to 5 seconds to ensure that it can complete 4 seconds of rotation without interruption.
+
+    .. code-block:: python
+        :linenos:
+
+        import iqmotion as iq
+        import time
+
+        #Create a SerialCommunicator object for our servo module
+        com = iq.SerialCommunicator("COM3")
+
+        # module = iq.SpeedModule(com, extra_clients=clients)
+        module = iq.SpeedModule(com)
+
+        #Set the timeout to a period of 5 seconds
+        module.set("propeller_motor_control", "timeout", 5)
+
+        #Set our Control Voltage to 1.5V and hold it for 4 seconds
+        module.set("propeller_motor_control", "ctrl_volts", 1.5)
+        time.sleep(4)
+
+        #Print our observed velocity
+        print("Observed velocity with ctrl_volts = 1.5V: ", round(module.get("brushless_drive", "obs_velocity"), 3), "rad/s")
+
+        #Set the module to coast, and wait 1 second
+        module.set("propeller_motor_control", "ctrl_coast")
+        time.sleep(1)
+
+        #Set our Control Voltage to -1V and hold it for 4 seconds
+        module.set("propeller_motor_control", "ctrl_pwm", 0.1)
+        time.sleep(4)
+
+        #Print our observed velocity
+        print("Observed velocity with observed supply voltage:", round(module.get("brushless_drive", "obs_supply_volts"), 3), "V",  round(module.get("brushless_drive", "obs_velocity"), 3), "rad/s")
+
+        #Coast the module
+        module.set("propeller_motor_control", "ctrl_coast")
+
+| The printed output:
+| *Observed velocity with ctrl_volts = 1.5V:  20.203 rad/s*
+| *Observed velocity with observed supply voltage: 14.929 V 20.495 rad/s*
+
+As expected, the observed velocities in both cases are practically identical.
+
+.. _tuning_parameters:
+
+Tuning Velocity Gains and Parameters
++++++++++++++++++++++++++++++++++++++++
+PID Tuning
+~~~~~~~~~~~~~~~~
+As mentioned :ref:`above <control_with_prop_motor_control>`, tuning the Propeller Motor Controller and Multi Turn Angle Controller differs due 
+to each controller's nature. For more information about tuning the Multi Turn Angle Controller please see :ref:`Position Control Tuning <position_control_tuning>`.
+
+Our velocity PID controller found in :ref:`Propeller Motor Controller <propeller_motor_control>` can be tuned as any standard PID controller. 
+With this controller, the proportional term acts on a velocity, our integral term on position, and our derivative term on acceleration. 
+
```

Comment:
The first section, "PID Tuning" sort of covers both Multi Turn and Propeller, linking elsewhere for multi turn and referring you below for propeller motor control. But if someone didn't read that section in its entirety, it may not be obvious these sections below PID Tuning are just for the Propeller Motor Controller. At least, I think they are just for it since they are in velocity units.

Maybe another heading here or something for "Propeller Motor Control PID Tuning" or something to that affect would help indicate that the stuff below is meant just for propeller motor control, not multi turn

Reply:
I've updated the PWM image to mention that it's also the full multi turn loop. There's not a great place otherwise to place it.